components/automatic-points-table/commons/automatic-points-table.lua

```diff
@@ -67,18 +69,43 @@ function AutomaticPointsTable.run(frame)
 	return divTable:create()
 end
 
+function AutomaticPointsTable:storeLPDB(pointsData)
+	Table.iter.forEachIndexed(pointsData, function(index, teamPointsData)
+		local team = teamPointsData.team
+		local teamName = string.lower(team.aliases[#team.aliases])
+		local lpdbName = self.parsedInput.lpdbName
+		local uniqueId = teamName .. '_' .. lpdbName
+		local position = teamPointsData.position
+		local totalPoints = teamPointsData.totalPoints
+		local objectData = {
+			type = lpdbName,

```

Comment:
It would be best to set `type` to something like `autopoints`. You can use the pagename to distinguish the various datapoints.

Reply:
```suggestion
			type = 'automatic points',
```